src/cider/type/plan/Analyzer.cpp

```diff
@@ -20,9 +20,12 @@
  * under the License.
  */
 #include "type/plan/Analyzer.h"
+#include "exec/nextgen/utils/JITExprValue.h"
 #include "exec/template/DateTimeUtils.h"
+#include "exec/template/IRCodegenUtils.h"
 #include "type/data/sqltypes.h"
```

Comment:
seems no need to include `JITExprValue.h`, also  `IRCodegenUtils.h` is old code path, better to migrate/rename useful func?

Reply:
Done.